### PR TITLE
Bump version of sbt, maven, and libraries

### PR DESCRIPTION
**Pull request by jiminhsieh**
I noticed after this commit https://github.com/akka/akka-samples/commit/f031f9ace0ddce49954e421d7852f06bf56ba10c, `pom.xml` was removed the profile of 2.11.x .Sample projects with maven will only work for 2.12.x. It means the flag, `-P`, of maven won't work.

### CODE REVIEW

**Comment by raboof** [comment]
Given the 'WIP' in the title, is there anything you want to do before merging?

File: akka-sample-camel-java/build.sbt [hunk 1]
```diff
@@ -2,6 +2,7 @@ organization := "com.typesafe.akka.samples"
 name := "akka-sample-camel-java"
 
 scalaVersion := "2.12.6"
+crossScalaVersions := Seq("2.12.6", "2.11.11")
```
Comment (patriknw) [inline akka-sample-camel-java/build.sbt]: I don't think we should complicate the build files of the samples by adding crossScalaVersions. That's an advanced feature that libraries but not applications may use. We want to promote latest version.

Comment (jiminhsieh) [reply akka-sample-camel-java/build.sbt]: The reason of adding crossScalaVersions is that build files without crossScalaVersions will throw:
```scala
java.lang.RuntimeException: Switch failed: no subprojects list "2.11.11" (or compatible version) in crossScalaVersions setting.
```
when executing:
```bash
sbt ++\$TRAVIS_SCALA_VERSION test
```

Or we follow the suggestion from sbt - If you want to force it regardless, call ++ 2.11.11!.

Comment (patriknw) [reply akka-sample-camel-java/build.sbt]: but shouldn't we remove Scala 2.11 from the Travis build instead?

Comment (jiminhsieh) [reply akka-sample-camel-java/build.sbt]: Got it. I will push soon. :)

**Pull request merged**